scripts/infra/README.md

```diff
@@ -49,6 +50,9 @@ Either way, you'll need to edit the config to reference your cloud environment(s
 Example:
 
 ```bash
+# Ensure proper system packages are installed if your local system is Red Hat-based
+# If you are running a different OS you'll need to install these packages manually
+./local-setup.sh rh ensure-aws-system-pkgs
```

Comment:
If this does install anything, you would have other steps to authenticate with aws still.  I am wondering whether it's best to just put a little more detail in Prereqs and not include the command here.

Reply:
Addressed in latest patchset